apps/frontend/src/app/5_pages/LeaderboardPointsPage/LeaderboardPointsPage.tsx

```diff
@@ -1,18 +1,18 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 
 import { t } from 'i18next';
 import { Helmet } from 'react-helmet-async';
 import { Trans } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 
-import { Button, ButtonSize, Heading, Link, Paragraph } from '@sovryn/ui';
+import { Button, ButtonSize, Heading, Paragraph, Tabs } from '@sovryn/ui';
 
 import { translations } from '../../../locales/i18n';
-import { CAMPAIGN_URL } from './LeaderboardPointsPage.constants';
-import { LeaderboardPointsFrame } from './components/LeaderboardPointsFrame/LeaderboardPointsFrame';
+import { TAB_ITEMS } from './LeaderboardPointsPage.constants';
 
 const LeaderboardPointsPage: FC = () => {
   const navigate = useNavigate();
+  const [tabIndex, setTabIndex] = useState(0);
 
   return (
```

Comment:
I think we need to adjust the title because it says "Extra Spice & Runes Season 1 Leaderboard" but we have tabs with 2 seasons there now.

<img width="2282" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/3738a7d8-37fd-4405-b0cc-1dbc6fe895c3">